src/pages/Task.tsx

```diff
@@ -270,12 +273,16 @@ const TaskCore: React.FC = () => {
                 >
                   <FilesTables />
                 </Tab>
-
                 {showBuildBaronTab ? (
                   <Tab name="Build Baron" id="task-build-baron-tab">
                     <BuildBaron data={buildBaronData} error={buildBaronError} />
                   </Tab>
                 ) : null}
+                {isPerfPluginEnabled && (
```

Comment:
Can you hardcode a false here so that the tab doesn't show until it's feature complete?